src/provider/soft.rs

```diff
@@ -28,6 +28,12 @@ use tracing::error;
 #[derive(Default)]
 pub struct SoftTpm {}
 
+impl SoftTpm {
+    pub fn new() -> Self {
+        SoftTpm::default()
```

Comment:
This one is a minor nit-pick. It was just annoying that `SoftTpm::new` disappeared between releases.

Reply:
Having a replica for `default()` is unnecessary since `::default`'s the rust idiomatic path for this kind of thing if you're not taking inputs/doing logic.

Reply:
I'm well aware of that, but the previous version of hsm-crypto provided `SoftTpm::new`.